### PR TITLE
x11-misc/urxvtconfig: New package

### PR DESCRIPTION
**Pull request by daedreth**
A graphical user interface tool for configuration of the rxvt-unicode terminal emulator.

### CODE REVIEW

File: x11-misc/urxvtconfig/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>eckertdawid@gmail.com</email>
+		<name>Dawid Eckert</name>
+	</maintainer>
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/metadata.xml]: Add:
```
	<maintainer type="project">
		<email>proxy-maint@gentoo.org</email>
		<name>Proxy Maintainers</name>
	</maintainer>
```

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+inherit qmake-utils
+
+DESCRIPTION="GUI configuration tool for the rxvt-unicode terminal emulator."
+HOMEPAGE="https://qithub.com/daedreth/URXVTConfig"
+
+SRC_URI="https://github.com/daedreth/URXVTConfig/archive/1.0.tar.gz -> urxvtconfig-1.0.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+MY_PN="${WORKDIR}/URXVTConfig-1.0"
+S="${MY_PN}"
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]:
```
MY_PN=URXVTConfig
S="${WORKDIR}/${MY_PN}-${PV}"
```

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	x11-terms/rxvt-unicode[xft]"
+
+DEPEND="${RDEPEND}
+	media-gfx/imagemagick
+	x11-libs/libXft
+	media-libs/fontconfig"
+
+DOCS=( README.md )
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: remove this... `->`

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+src_prepare() {
+	default
+}
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: remove this whole function

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+src_configure() {
+	eqmake5 source/URXVTConfig.pro
+}
+
+src_install() {
+	emake INSTALL_ROOT="${D}" install
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: `->` ...and add here `einstalldocs`

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="GUI configuration tool for the rxvt-unicode terminal emulator."
+HOMEPAGE="https://qithub.com/daedreth/URXVTConfig"
+
+SRC_URI="https://github.com/daedreth/URXVTConfig/archive/1.0.tar.gz -> urxvtconfig-1.0.tar.gz"
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: `SRC_URI="https://github.com/daedreth/URXVTConfig/archive/1.0.tar.gz -> ${P}.tar.gz"`

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="GUI configuration tool for the rxvt-unicode terminal emulator."
+HOMEPAGE="https://qithub.com/daedreth/URXVTConfig"
+
+SRC_URI="https://github.com/daedreth/URXVTConfig/archive/1.0.tar.gz -> ${P}.tar.gz"
```
Comment (a17r) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: `...archive/${PV}.tar.gz` if you want to have even less work on your next bump.

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="GUI configuration tool for the rxvt-unicode terminal emulator."
+HOMEPAGE="https://github.com/daedreth/URXVTConfig"
+
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: remove this empty line

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 9]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: put `MY_PN=URXVTConfig` here

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+DESCRIPTION="GUI configuration tool for the rxvt-unicode terminal emulator."
+HOMEPAGE="https://github.com/daedreth/URXVTConfig"
+
+SRC_URI="https://github.com/daedreth/URXVTConfig/archive/${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: `SRC_URI="https://github.com/daedreth/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"`

File: x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+KEYWORDS="~amd64 ~x86"
+
+MY_PN=URXVTConfig
```
Comment (SoapGentoo) [inline x11-misc/urxvtconfig/urxvtconfig-1.0.ebuild]: remvoe this one

**Pull request closed without merging**